commit 1f6265e011f6e56562b2f58c182bc0261062b3c4
Author: Brad <brad@comstyle.com>
Date:   Wed Apr 6 17:41:12 2011 -0400

    Fix parser: mark av_parser_parse() for removal on next major bump
    
    Fix the breakage introduced with 9073ca6c6ebb9e3e9b60d968b7aa16cc8b6d558d

diff --git a/libavcodec/parser.c b/libavcodec/parser.c
index 0c69b91..3ba3e9c 100644
--- a/libavcodec/parser.c
+++ b/libavcodec/parser.c
@@ -105,7 +105,7 @@ void ff_fetch_timestamp(AVCodecParserContext *s, int off, int remove){
     }
 }
 
-#if LIBAVCODEC_VERSION_MINOR < 53
+#if LIBAVCODEC_VERSION_MAJOR < 53
 /**
  *
  * @param buf           input
